lib/ziti_tunneler.c

```diff
@@ -136,13 +143,13 @@ void ziti_tunneler_stop_intercepting(tunneler_context tnlr_ctx, const char *serv
 }
 
 /** called by tunneler application when data is read from a ziti connection */
-int ziti_tunneler_write(tunneler_io_context *tnlr_io_ctx, const void *data, size_t len) {
+ssize_t ziti_tunneler_write(tunneler_io_context *tnlr_io_ctx, const void *data, size_t len) {
```

Comment:
I think using ssize_t isn't particularly windows friendly. @ekoby, correct?

Reply:
correct in general, but I both libuv and lwip declare proper type and we get it